server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const app = express()
+
+
+//routes
+
+app.get('/', (req, res) => {
+    res.sendStatus("Hello NODE API")
+})
+
+
+app.listen(5100, ()=> {
+    console.log("Node API app is running on port 3000")
```

Comment:
The console log message does not match the port the server is listening on. It's important to keep these consistent to avoid confusion. [medium]